packages/codegen-ui/lib/types/studio-types.ts

```diff
@@ -320,14 +320,19 @@ export type ConcatenatedStudioComponentProperty = {
   concat: StudioComponentProperty[];
 };
 
+/**
+ * These are the string values accepted by operator
+ */
+export type Operator = 'eq' | 'ne' | 'le' | 'lt' | 'ge' | 'gt';
```

Comment:
Could you update to a slightly more specific name, like 'ConditionalOperator'

Reply:
Good point, but to be specific - How about this name `Comparator` ?

Reply:
Went with `RelationalOperator`